packages/nimble-components/src/select/styles.ts

```diff
@@ -28,6 +40,81 @@ export const styles = css`
     [part='end'] {
         display: contents;
     }
+
+    .listbox {
+        overflow-x: clip;
+    }
+
+    .listbox.empty slot {
+        display: none;
+    }
+
+    .listbox.above {
+        flex-flow: column-reverse;
+    }
+
+    .search-field {
+        display: flex;
+        flex-direction: row;
+        align-items: center;
+        height: ${controlHeight};
+        background: transparent;
+    }
+
+    .search-icon {
+        padding-left: ${smallPadding};
+    }
+
+    .search-field::after,
+    .search-field::before {
+        content: '';
+        position: absolute;
+        height: 0px;
+        border-bottom: rgba(${borderRgbPartialColor}, 0.15) 2px solid;
+    }
+
+    .search-field::after {
+        top: calc(${controlHeight} + ${smallPadding} - ${borderWidth});
+    }
+
+    .search-field:before {
+        bottom: calc(${controlHeight} + ${smallPadding} + (2 * ${borderWidth}));
+    }
+
+    .search-field:not(.above)::after,
+    .search-field.above::before {
+        width: calc(100% - (2 * ${borderWidth}));
+    }
```

Comment:
A few questions here:
1. Why can't this `width` styling go above in the larger block of shared `.search-field::after, .search-field::before` styling?
2. It looks like this is attempting to make the gray divider line not span the entire width of the drop-down. Is that correct? It looks that way in the visual designs, but I don't see that being the case in storybook.